corehq/apps/smsbillables/migrations/0005_smsbillable_twilio_gateway_fee.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('smsbillables', '0004_bootstrap_smsgh_rates'),
+    ]
+
+    operations = [
```

Comment:
This involves adding a column to a table of more than 3 million rows.  Will it be a problem if the table has to be locked in what will be a long-running migration?